<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>395</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QTextBrowser" name="tb_rx_buf">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>接收窗口</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>16进制接收</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_clear">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>清除接收</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTextEdit" name="te_tx_buf">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>发送窗口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>16进制发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_send_enter">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>发送新行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_send">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>单命令行发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>波特率：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbb_baud"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>串口号：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbb_com"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_serch">
         <property name="text">
          <string>搜索串口</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_uart_Ctrl">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lbe_red">
         <property name="text">
          <string>机器开关</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_led1_Ctrl">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>唤醒</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbe_blue">
         <property name="text">
          <string>测试图片</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>ColorBars</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>White</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn_led2_Ctrl">
         <property name="text">
          <string>确认测试图</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="btn_led1_Ctrl_2">
         <property name="text">
          <string>发送界面当前配置</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>保存界面当前配置</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>配置界面显示</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>自动测试界面显示</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>显示实时数据图像</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>开始保存数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
